aiohttp/web.py

```diff
@@ -277,6 +280,130 @@
 
 HostSequence = TypingIterable[str]
 
+if sys.version_info >= (3, 11):
+
+    class _State(enum.Enum):
+        CREATED = "created"
+        INITIALIZED = "initialized"
+        CLOSED = "closed"
+
+    class WebRunner(asyncio.Runner):  # type: ignore
```

Comment:
We should probably get a new test that uses WebRunner directly too. Can probably just copy and modify one of the run_app() tests. The new test could also use WebRunner.run() to verify that the task and the app are run in the same loop.